### PR TITLE
CRM457-1457: Export metrics to prometheus

### PR DESCRIPTION
**Pull request by patrick-laa**
## Description of change

 [Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1457)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ gem 'logstasher', '~> 2.1'
 gem 'oauth2', '~> 2.0'
 gem 'pagy', '~> 8.4.2'
 gem 'pg', '~> 1.5'
-gem 'prometheus-client'
+gem 'prometheus_exporter'
```
Comment (jogilder) [inline Gemfile:20]: why does this not have prometheus-client but the gem lock file does?

File: Gemfile.lock [hunk 2]
```diff
@@ -312,6 +312,8 @@ GEM
       optimist (~> 3.0)
     pg (1.5.6)
     prometheus-client (2.1.0)
```
Comment (jogilder) [inline Gemfile.lock:314]: still has prometheus-client - is this a dependency?

**Review by jogilder — COMMENTED** [review]
see comment below

File: Gemfile.lock (hunk 2 above)
Comment (patrick-laa) [reply Gemfile.lock:314]: exactly - sidekiq_prometheus has a dependency on prometheus-client

File: Gemfile (hunk 1 above)
Comment (jogilder) [reply Gemfile:20]: prometheus client for sidekiq work in another ticket hence needed here.

**Review by jogilder — APPROVED** [review]
LGTM

**Pull request merged**